### PR TITLE
ResNet-50 || ImageNet Training on MXNet || ModelArts Service of Huawei Cloud

### PR DESCRIPTION
**Pull request by TccccD**
Compared to the last time, we mainly updated the following content：

1. Using nccl 2.1 on MXNet;
2. Use the memory warmup mode to avoid the memory overflow caused by nccl taking up too much memory under MXNet;

@codyaustun please check it.

### CODE REVIEW

File: ImageNet/train/huaweicloud_resnet50_modelarts_16_machine_mxnet121.json
Comment (codyaustun) [inline ImageNet/train/huaweicloud_resnet50_modelarts_16_machine_mxnet121.json]: Because you only report 30 epochs in the TSV, I think this should be 2.

File: ImageNet/train/huaweicloud_resnet50_modelarts_16_machine_mxnet121.json
Comment (codyaustun) [inline ImageNet/train/huaweicloud_resnet50_modelarts_16_machine_mxnet121.json]: Same as above

**Review by codyaustun — APPROVED** [review]
This looks good to me. Let me know if you want to update the number of epochs for the last segments of the `imageSize` and `batchSize` schedules. If you do, I'll wait until the JSON file is updated to merge in this pull request.

File: ImageNet/train/huaweicloud_resnet50_modelarts_16_machine_mxnet121.json
Comment (TccccD) [reply ImageNet/train/huaweicloud_resnet50_modelarts_16_machine_mxnet121.json]: Oh, this is my negligence, I updated it.
@codyaustun

Comment (codyaustun) [reply ImageNet/train/huaweicloud_resnet50_modelarts_16_machine_mxnet121.json]: Thanks!

File: ImageNet/train/huaweicloud_resnet50_modelarts_16_machine_mxnet121.json
Comment (codyaustun) [inline ImageNet/train/huaweicloud_resnet50_modelarts_16_machine_mxnet121.json:6]: @TccccD sorry I noticed you deleted the `codeURL`. Was that intended?

Comment (TccccD) [reply ImageNet/train/huaweicloud_resnet50_modelarts_16_machine_mxnet121.json:6]: Oh, ok...... We will update later

**Pull request merged**
